setup.py

```diff
@@ -0,0 +1,46 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from setuptools import setup, find_packages
+
+with open(Path(__file__).parent / 'LICENSE') as f:
+    license_ct = f.read()
+with open(Path(__file__).parent / 'wrappers' / 'python3' / 'VERSION') as f:
+    version = f.read()
```

Comment:
To get the version do this instead:
```python
import eHive
version = eHive.__version__
```
No need to add the `VERSION` file (and those going out of sync)

Reply:
That was the idea I had first. but didn't work when running setup upon pip install. I'll try again today.

Reply:
I mean eHive is not in "PATH" when unpacking compressed built library.

Reply:
Right, so
```python
from wrappers.python3 import eHive
```
should work, but it doesn't yet. I will add another commit to the PR to address this and the version there